diff --git a/dist/index.cjs b/dist/index.cjs
index 873623560be9af95a3e22534a25c058abff0840c..6411346216603589e9844e0c9f3899dc0a48c378 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -100,7 +100,7 @@ var ExtraConfigColumns = Symbol.for("drizzle:ExtraConfigColumns");
 var import_pg_core = require("drizzle-orm/pg-core");
 var import_mysql_core = require("drizzle-orm/mysql-core");
 var import_sqlite_core = require("drizzle-orm/sqlite-core");
-var import_casing = require("drizzle-orm/casing");
+
 var import_fs = require("fs");
 var import_path = require("path");
 var BaseGenerator = class {
@@ -112,7 +112,7 @@ var BaseGenerator = class {
     escapeName: () => "",
     escapeParam: () => "",
     escapeString: () => "",
-    casing: new import_casing.CasingCache()
+    casing: {}
   };
   constructor(schema, relational) {
     this.schema = schema;
@@ -315,7 +315,7 @@ var BaseGenerator = class {
         generatedEnums.push(this.generateEnum(value));
       } else if ((0, import_drizzle_orm.is)(value, import_pg_core.PgTable) || (0, import_drizzle_orm.is)(value, import_mysql_core.MySqlTable) || (0, import_drizzle_orm.is)(value, import_sqlite_core.SQLiteTable)) {
         generatedTables.push(this.generateTable(value));
-      } else if ((0, import_drizzle_orm.is)(value, import_drizzle_orm.Relations)) {
+      } else if (import_drizzle_orm.Relations && (0, import_drizzle_orm.is)(value, import_drizzle_orm.Relations)) {
         relations.push(value);
       }
     }
@@ -337,14 +337,14 @@ function writeDBMLFile(dbml, outPath) {
 }
 
 // src/generators/pg.ts
-var import_casing2 = require("drizzle-orm/casing");
+
 var PgGenerator = class extends BaseGenerator {
   InlineForeignKeys = PgInlineForeignKeys;
   buildQueryConfig = {
     escapeName: (name) => `"${name}"`,
     escapeParam: (num) => `$${num + 1}`,
     escapeString: (str) => `'${str.replace(/'/g, "''")}'`,
-    casing: new import_casing2.CasingCache()
+    casing: {}
   };
   isIncremental(column) {
     return column.getSQLType().includes("serial");
@@ -368,14 +368,14 @@ function pgGenerate(options) {
 // src/generators/mysql.ts
 var import_drizzle_orm2 = require("drizzle-orm");
 var import_mysql_core2 = require("drizzle-orm/mysql-core");
-var import_casing3 = require("drizzle-orm/casing");
+
 var MySqlGenerator = class extends BaseGenerator {
   InlineForeignKeys = MySqlInlineForeignKeys;
   buildQueryConfig = {
     escapeName: (name) => `\`${name}\``,
     escapeParam: (_num) => "?",
     escapeString: (str) => `'${str.replace(/'/g, "''")}'`,
-    casing: new import_casing3.CasingCache()
+    casing: {}
   };
   isIncremental(column) {
     return column.getSQLType().includes("serial") || (0, import_drizzle_orm2.is)(column, import_mysql_core2.MySqlColumnWithAutoIncrement) && column.autoIncrement;
@@ -391,14 +391,14 @@ function mysqlGenerate(options) {
 // src/generators/sqlite.ts
 var import_drizzle_orm3 = require("drizzle-orm");
 var import_sqlite_core2 = require("drizzle-orm/sqlite-core");
-var import_casing4 = require("drizzle-orm/casing");
+
 var SQLiteGenerator = class extends BaseGenerator {
   InlineForeignKeys = SQLiteInlineForeignKeys;
   buildQueryConfig = {
     escapeName: (name) => `"${name}"`,
     escapeParam: (_num) => "?",
     escapeString: (str) => `'${str.replace(/'/g, "''")}'`,
-    casing: new import_casing4.CasingCache()
+    casing: {}
   };
   isIncremental(column) {
     return (0, import_drizzle_orm3.is)(column, import_sqlite_core2.SQLiteBaseInteger) && column.autoIncrement;
diff --git a/dist/index.js b/dist/index.js
index 0bcb37a4a8fe8755ffc6aee42e84ac8df0070787..4894d8d751904bc13eadcf22ddf17436c4470652 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -103,7 +103,6 @@ import {
   UniqueConstraint as SQLiteUniqueConstraint,
   Check as SQLiteCheck
 } from "drizzle-orm/sqlite-core";
-import { CasingCache } from "drizzle-orm/casing";
 import { writeFileSync } from "fs";
 import { resolve } from "path";
 var BaseGenerator = class {
@@ -115,7 +114,7 @@ var BaseGenerator = class {
     escapeName: () => "",
     escapeParam: () => "",
     escapeString: () => "",
-    casing: new CasingCache()
+    casing: {}
   };
   constructor(schema, relational) {
     this.schema = schema;
@@ -318,7 +317,7 @@ var BaseGenerator = class {
         generatedEnums.push(this.generateEnum(value));
       } else if (is(value, PgTable) || is(value, MySqlTable) || is(value, SQLiteTable)) {
         generatedTables.push(this.generateTable(value));
-      } else if (is(value, Relations)) {
+      } else if (typeof Relations !== "undefined" && is(value, Relations)) {
         relations.push(value);
       }
     }
@@ -340,14 +339,13 @@ function writeDBMLFile(dbml, outPath) {
 }
 
 // src/generators/pg.ts
-import { CasingCache as CasingCache2 } from "drizzle-orm/casing";
 var PgGenerator = class extends BaseGenerator {
   InlineForeignKeys = PgInlineForeignKeys;
   buildQueryConfig = {
     escapeName: (name) => `"${name}"`,
     escapeParam: (num) => `$${num + 1}`,
     escapeString: (str) => `'${str.replace(/'/g, "''")}'`,
-    casing: new CasingCache2()
+    casing: {}
   };
   isIncremental(column) {
     return column.getSQLType().includes("serial");
@@ -371,14 +369,13 @@ function pgGenerate(options) {
 // src/generators/mysql.ts
 import { is as is2 } from "drizzle-orm";
 import { MySqlColumnWithAutoIncrement } from "drizzle-orm/mysql-core";
-import { CasingCache as CasingCache3 } from "drizzle-orm/casing";
 var MySqlGenerator = class extends BaseGenerator {
   InlineForeignKeys = MySqlInlineForeignKeys;
   buildQueryConfig = {
     escapeName: (name) => `\`${name}\``,
     escapeParam: (_num) => "?",
     escapeString: (str) => `'${str.replace(/'/g, "''")}'`,
-    casing: new CasingCache3()
+    casing: {}
   };
   isIncremental(column) {
     return column.getSQLType().includes("serial") || is2(column, MySqlColumnWithAutoIncrement) && column.autoIncrement;
@@ -394,14 +391,13 @@ function mysqlGenerate(options) {
 // src/generators/sqlite.ts
 import { is as is3, SQL as SQL2 } from "drizzle-orm";
 import { SQLiteBaseInteger } from "drizzle-orm/sqlite-core";
-import { CasingCache as CasingCache4 } from "drizzle-orm/casing";
 var SQLiteGenerator = class extends BaseGenerator {
   InlineForeignKeys = SQLiteInlineForeignKeys;
   buildQueryConfig = {
     escapeName: (name) => `"${name}"`,
     escapeParam: (_num) => "?",
     escapeString: (str) => `'${str.replace(/'/g, "''")}'`,
-    casing: new CasingCache4()
+    casing: {}
   };
   isIncremental(column) {
     return is3(column, SQLiteBaseInteger) && column.autoIncrement;
